### PR TITLE
Make organization names as uniform as possible.

### PR DESCRIPTION
**Pull request by bbockelm**

### CODE REVIEW

**Comment by djw8605** [comment]
Great minds think alike #562 ?

**Comment by bbockelm** [comment]
Oh, you got some I missed.  Pulled those into the branch.

**Comment by ChristinaLK** [comment]
Why are some of the UC schools `University of California, location` vs `University of California Location`?

**Comment by bbockelm** [comment]
@ChristinaLK - I checked the respective homepages.  Seems to be a quirk of UCSD versus all the other UC schools.

**Comment by matyasselmeci** [comment]
There's a "_Las_ Alamos National Lab" in there; might as well fix that up while we're at it

**Comment by matyasselmeci** [comment]
We should also make sure that organizations that are also facilities in `topology/` have matching names between projects and topology.

File: projects/HCCLocalSubmit.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ Description: Jobs from our clusters that flock to the OSG
 FieldOfScience: Community Grid
 ID: '363'
 Name: HCCLocalSubmit
-Organization: University of Nebraska
+Organization: University of Nebraska-Lincoln
```
Comment (CarrieBrown) [inline projects/HCCLocalSubmit.yaml]:
```suggestion
Organization: University of Nebraska
```

**Comment by bbockelm** [comment]
@matyasselmeci - you are correct, but I think it should be separate from this PR.  Let's put that one into a JIRA ticket...

**Comment by brianhlin** [comment]
Does that mean commas in facility names? Ugh

**Comment by bbockelm** [comment]
@brianhlin - I did manage to remove all the unicode dashes!

File: projects/PBOSD.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ Description: "Probability-based structural design has now emerged as the most ad
 FieldOfScience: Civil Engineering
 ID: '364'
 Name: PBOSD
-Organization: UC San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/PBOSD.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/PCFOSGUCSD.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ Description: Work submitted as part of the physics computing facility (PCF) at t
   physics department at UCSD
 FieldOfScience: Physics
 ID: '369'
-Name: PCFOSGUCSD
+Name: University of California San Diego
```
Comment (CarrieBrown) [inline projects/PCFOSGUCSD.yaml]:
```suggestion
Name: University of California, San Diego
```

File: projects/POLARBEAR.yaml [hunk 4]
```diff
@@ -11,7 +11,7 @@ Description: The evolution of the universe is based on the idea of gravitational
 FieldOfScience: Astrophysics
 ID: '334'
 Name: POLARBEAR
-Organization: University of California - San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/POLARBEAR.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/Phylo.yaml [hunk 5]
```diff
@@ -6,7 +6,7 @@ Description: In this project, we design, implement, and test new methods for phy
 FieldOfScience: Bioinformatics
 ID: '178'
 Name: Phylo
-Organization: UC San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/Phylo.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/TG-STA110014S.yaml [hunk 6]
```diff
@@ -3,7 +3,7 @@ Description: Staff Account for Training
 FieldOfScience: Training
 ID: '48'
 Name: TG-STA110014S
-Organization: University of California-San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/TG-STA110014S.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/UCSDEngEarthquake.yaml [hunk 7]
```diff
@@ -3,7 +3,7 @@ Description: Earthquake Engineering from UCSD supported users
 FieldOfScience: Engineering
 ID: '81'
 Name: UCSDEngEarthquake
-Organization: UCSD
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/UCSDEngEarthquake.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/UCSDPhysAstroExp.yaml [hunk 8]
```diff
@@ -3,7 +3,7 @@ Description: Experimental Astrophysics from UCSD supported users
 FieldOfScience: Astrophysics
 ID: '80'
 Name: UCSDPhysAstroExp
-Organization: UCSD
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/UCSDPhysAstroExp.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/UCSDPhysAstroTheo.yaml [hunk 9]
```diff
@@ -3,7 +3,7 @@ Description: Theoretical Astrophysics from UCSD supported users
 FieldOfScience: Astrophysics
 ID: '79'
 Name: UCSDPhysAstroTheo
-Organization: UCSD
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/UCSDPhysAstroTheo.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/UCSDPhysBio.yaml [hunk 10]
```diff
@@ -3,7 +3,7 @@ Description: Biological Physics from UCSD supported users
 FieldOfScience: Biophysics
 ID: '83'
 Name: UCSDPhysBio
-Organization: UCSD
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/UCSDPhysBio.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/UCSDPhysPart.yaml [hunk 11]
```diff
@@ -3,7 +3,7 @@ Description: Non-CMS Particle Physics from UCSD supported users
 FieldOfScience: High Energy Physics
 ID: '82'
 Name: UCSDPhysPart
-Organization: UCSD
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/UCSDPhysPart.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/cms.org.ucsd.yaml [hunk 12]
```diff
@@ -3,7 +3,7 @@ Description: CMS Connect group for UCSD
 FieldOfScience: High Energy Physics
 ID: '141'
 Name: cms.org.ucsd
-Organization: University of California, San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/cms.org.ucsd.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/nnmbl.yaml [hunk 13]
```diff
@@ -7,7 +7,7 @@ Description: I will be using techniques in machine learning and AI to
   neural nets.
 Department: Physics
 FieldOfScience: Computational Condensed Matter Physics
-Organization: UC San Diego
+Organization: University of California San Diego
```
Comment (CarrieBrown) [inline projects/nnmbl.yaml]:
```suggestion
Organization: University of California, San Diego
```

File: projects/PBOSD.yaml (hunk 2 above)
Comment (matyasselmeci) [reply projects/PBOSD.yaml]: From ucsd.edu, it looks like there's no comma in the name.

Comment (matyasselmeci) [reply projects/PBOSD.yaml]: Never mind. They have a comma in their LinkedIn page.

**Pull request merged**